package.json

```diff
@@ -27,6 +27,10 @@
     "lint": "eslint --cache --fix .",
     "lint:check": "eslint --cache .",
     "lint:links": "iota-wiki check",
+    "lint:md": "yarn iota-wiki lint -c .markdownlint-cli2.cjs",
+    "lint:md:check:iota": "yarn iota-wiki lint iota/**/*.{md,mdx}",
+    "lint:md:check:shimmer": "yarn iota-wiki lint shimmer/**/*.{md,mdx}",
+    "lint:md:check:next": "yarn iota-wiki lint next/**/*.{md,mdx}",
```

Comment:
Somehow these commands still fixes fixable problems, even though your setup of config files seems to imply you don't want that.

Reply:
Also I think we should ignore the `external` directories, and check linting rules in their own repos just like we do with the formatting. Wdyt?

Reply:
> Somehow these commands still fix fixable problems, even though your setup of config files seems to imply you don't want that.

Will check

> Also I think we should ignore the external directories, and check linting rules in their own repos just like we do with the formatting. Wdyt?

Yes, we could also update the cli in the TIPS repo if we really want to

Reply:
I couldn't fix it. But as I had to update `markdownlint-cli2` anyway, I did and that fixed it it seems 😆